src/core/TimeTraveller.js

```diff
@@ -77,6 +77,7 @@ const TimeTraveller = types
       },
 
       onUpdate(handler) {
+        this.lastAdditionTime = new Date();
```

Comment:
does the lastAdditionTime get saved to the annotation? if so there could be issues around users from differnt time zones interacting with it.
aside from that the concept makes sense and im ready to approve - just wondering about this

Reply:
it does not,  good thinking tho